deployment/ansible/roles/cac-tripplanner.app/templates/upstart-cac-tripplanner-app.conf.j2

```diff
@@ -0,0 +1,14 @@
+description "Gunicorn upstart script for CAC Trip Planner"
+
+{% if develop or test -%}
+start on (vagrant-mounted)
+{% else %}
+start on (local-filesystems and net-device-up IFACE!=lo)
+{% endif %}
+stop on shutdown
+
+respawn
+setuid {{ app_username }}
+chdir {{ root_app_dir }}
+
+exec gunicorn --config {{ root_conf_dir }}/gunicorn.py cac_tripplanner.wsgi
```

Comment:
Yep, there isn't one right now. Not a bad idea, but as Chris said, it would need to expose a bunch of the attributes so that people could override them for their specific needs.

Also, here is a list of the existing roles: https://galaxy.ansible.com/list#/users/6541